### ISSUE TITLE
Windows Vista Client unable to fetch updates from LegacyUpdate server

### ISSUE THREAD

**Issue by reiyua** [LegacyUpdate/LegacyUpdate#245]
Hello!

I've been using LegacyUpdate a lot recently to get some old Windows laptops going from ages ago and has been working VERY well until I attempted to run your software on an old Windows Vista based Compac Laptop named "Presario V6500 Notebook PC", specifications below:
![PC Specs](https://github.com/LegacyUpdate/LegacyUpdate/assets/11124303/c394d2f6-d5d5-4117-b200-6fed9be77aef)

(Not entirely sure if this user manual will help at all, but is the ONLY information I could find for it online: http://h10032.www1.hp.com/ctg/Manual/c01078025.pdf )

The overall installer when installing things such as Root Certificate Updates and Service Packs did have some issues aswell, but they seem to be easily resolved by a simple reload of the setup file and continuing the install that way.

After completing the installer, all I see is this:
![LegacyUpdate loading forever](https://github.com/LegacyUpdate/LegacyUpdate/assets/11124303/274b890a-0f3d-433f-b276-2c11f6f0eeea)

Not sure if the Update History would help, but here is a screenshot of it:
![Update History](https://github.com/LegacyUpdate/LegacyUpdate/assets/11124303/ba00b691-93ce-42d8-9369-751b0a745673)

This was completed using version 1.9 of the software.

Cheers in advance for any help you can provide!

**Comment by LazyNerdYT** [comment]
on vista it just takes a very long time, just let it run for awhile

**Comment by reiyua** [comment]
I let it run for 4 hours, maybe even a little bit more, how long did it take for yours to tick over? @LazyNerdYT

**Comment by LazyNerdYT** [comment]
join our discord for more help

**Comment by reiyua** [comment]
Good idea, shall do it now 👍

**Comment by reiyua** [comment]
Also forgot to mention, Windows Update. o longer works either, just has the same issue as the main LegacyUpdate homepage.

**Comment by reiyua** [comment]
I'm gonna close this support ticket, surprisingly, using the in-built Windows Update installer instead of the LegacyUpdate website WITH LegacyUpdate installed works fine... 

I have a feeling it modified the update servers in the end

TLDR: Problem solved :)
